apps/web/pages/api/user/avatar.ts

```diff
@@ -64,6 +65,28 @@ async function getIdentityData(req: NextApiRequest) {
       avatar: team?.logo || getPlaceholderAvatar(null, teamname),
     };
   }
+  if (orgSlug) {
+    const org = await prisma.team.findFirst({
+      where: {
+        slug: orgSlug,
```

Comment:
You would need to check `requestedSlug` as well. See, the usage of `getSlugOrRequestedSlug` above